torb/tests/test_beanstalk_utils.py

```diff
@@ -64,7 +64,7 @@ def test_powerup_logs_to_foursight(waitfor_json):
 def test_wait_for_travis():
     input_json = {
             "dry_run": False,
-            "id": 312587257,
+            "id": 312587257,  # This 'id' needs a better explanatory comment. -kmp 10-Apr-2020
```

Comment:
I don't think the ID itself here is meaningful. Probably just randomly selected.

Reply:
Well, I wanted to research whether it either is a thing that's garbage or if it's something that's a known-finished one that is not being waited on. Can do it later. I'll leave the comment for now. Just can't take time out for everything.